test/tasks/hello-world/1.0/hello-world_test.go

```diff
@@ -43,9 +45,9 @@ func TestTaskHelloWorld(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 
 			framework.Run(t, tc, framework.TestOpts{
-				TaskKindRef:   "Task",
-				TaskName:      "hello-world",
-				WorkspaceName: "source", // must exist in the Task definition
+				TaskKindRef:   "Task",        // could be read from task definition
+				TaskName:      "hello-world", // could be read from task definition
+				WorkspaceName: "source",      // must exist in the Task definition, could be read
```

Comment:
@michaelsauter Good point, I guess we can create a helper func that unmarshalls the content of the Task YAML file into a `tekton.Task` object. I like it :)